Framework/src/TaskRunner.cxx

```diff
@@ -455,7 +455,7 @@ void TaskRunner::startOfActivity()
 
   // register ourselves to the BK
   if (gSystem->Getenv("O2_QC_REGISTER_IN_BK")) { // until we are sure it works, we have to turn it on
-    ILOG(Debug, Devel) << "Registering taskRunner to BookKeeping" << ENDM;
+    ILOG(Info, Devel) << "Registering taskRunner to BookKeeping" << ENDM;
```

Comment:
i recommend using `git add -p` to individually accept each change you want bring to the commit ;)

Reply:
thank you for your suggestion :-D